pkg/scheduler/framework/session_plugins.go

```diff
@@ -800,7 +800,14 @@ func (ssn *Session) BuildVictimsPriorityQueue(victims []*api.TaskInfo) *util.Pri
 		if lv.Job == rv.Job {
 			return !ssn.TaskOrderFn(l, r)
 		}
-		return !ssn.JobOrderFn(ssn.Jobs[lv.Job], ssn.Jobs[rv.Job])
+
+		lvJob, lvJobFound := ssn.Jobs[lv.Job]
```

Comment:
We'd beeter add ut here too.

Reply:
I tried to write this UT and found that the internal logic of the BuildVictimsPriorityQueue method is implemented within Plugins. The session_plugins_test seems more like something at the framework level, and it feels a bit odd to test with specific Plugins logic here (it would also cause circular references). Additionally, this testing logic appears to be the same as the logic in the UT for reclaim.

Reply:
OK